src/pages/Quiz.jsx

```diff
@@ -17,6 +21,15 @@ export const Quiz = () => {
     setQuestionPage(questionPage + 1);
   };
 
+  const QUESTIONCOUNT = reactRules.length;
```

Comment:
I'm not sure that it should be a separate variable, because this value will be different for every quiz.

And if it's necessary and you explain why it should be here, `QUESTION_QTY` will be more semantic and convention naming for it, IMO. 🙂

Reply:
done